.github/workflows/checks.yml

```
@@ -201,14 +201,20 @@ jobs:
           node-version-file: .nvmrc
       - name: Install dependencies
         run: npm clean-install
-      - name: Transpile to CommonJS
-        run: npm run transpile
+      - name: Simulate publish
+        run: |
+          npm publish --dry-run
+          npm pack
```

I feel like it might make sense to explain what is being done here:

```suggestion
      - name: Simulate publish
        run: |
          # Dry run publish to trigger any hooks
          npm publish --dry-run
          
          # Pack to produce the archive that would be published
          npm pack
```